lib/main.dart

```diff
@@ -68,15 +127,147 @@ class _MyHomePageState extends State<MyHomePage> {
+  // Функция для вставки видео в плейер
+  @override
+  void initState() {
+    super.initState();
+    // Play a [Media] or [Playlist].
+    player.open(Media('https://user-images.githubusercontent.com/28951144/229373695-22f88f13-d18f-4288-9bf1-c3e078d83722.mp4'));
+  }
+
+  @override
+  void dispose() {
+    player.dispose();
+    super.dispose();
+  }
+
+  void _tellTime() {
+    ScaffoldMessenger.of(context).showSnackBar(SnackBar(
+       content: Text("Video is on ${player.state.position}"),
+     ));
+    //print(player.state.position);
+  }
+
   @override
   Widget build(BuildContext context) {
-    // This method is rerun every time setState is called, for instance as done
-    // by the _incrementCounter method above.
-    //
-    // The Flutter framework has been optimized to make rerunning build methods
-    // fast, so that you can just rebuild anything that needs updating rather
-    // than having to individually change instances of widgets.
+
     return Scaffold(
+      appBar: AppBar( //Верхняя часть с именем
+        // TRY THIS: Try changing the color here to a specific color (to
+        // Colors.amber, perhaps?) and trigger a hot reload to see the AppBar
+        // change color while the other colors stay the same.
+        backgroundColor: Theme.of(context).colorScheme.inversePrimary,
+        // Here we take the value from the MyHomePage object that was created by
+        // the App.build method, and use it to set our appbar title.
+        title: Text(widget.title),
+      ),
+      body: 
+      Row( //Тело, разделённое по колонкам
+      children: [
+        
+        Column(children: [
+          
+          SizedBox( // Коробка под видео
+            width: MediaQuery.sizeOf(context).width * 0.7,
+            //width: MediaQuery.of(context).size.width,
+            height: MediaQuery.of(context).size.width * 0.7 * 9.0 / 16.0,
+            // Use [Video] widget to display video output.
+            child: Video(controller: controller),
+          ),
+
+          Row( // Для кнопок нужно разделить пространство по столбикам
+            children: [
+
+              SizedBox(
+                width: 120,
+                height: 120,
+                child: FloatingActionButton(
+                  onPressed: _tellTime,
+                  tooltip: 'Tells the time',
+                  child: const Icon(Icons.access_time_outlined),
+                ),
+              ),
+
+              SizedBox(
+                width: 120,
+                height: 120,
+                child: FloatingActionButton(
+                  onPressed: getFileVideo,
+                  tooltip: 'Choose video file',
+                  child: const Icon(Icons.video_call_rounded),
+                ),
+              ),
+
+              SizedBox(
+                width: 120,
+                height: 120,
+                child: FloatingActionButton(
+                  onPressed: getFileSubtitle,
+                  tooltip: 'Choose video file',
```

Comment:
Так, стоп, вряд ли эти две кнопки одинаковы. Может быть...?
```suggestion
                  tooltip: 'Choose video file',
                  child: const Icon(Icons.video_call_rounded),
                ),
              ),

              SizedBox(
                width: 120,
                height: 120,
                child: FloatingActionButton(
                  onPressed: getFileSubtitle,
                  tooltip: 'Choose subtitle file',
```